index.html

```diff
@@ -73,6 +76,196 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
+              <li>2015</li>
+            </ul>
+            <p class="work_details_content">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work_cats">
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+
+        <!-- work card 4-->
+        <div class="work_card">
+          <div class="work_preview">
+            <img src="./images/snapshoot-portfolio4.png" alt="Project 4" />
+          </div>
+          <div class="work_details">
+            <h1 class="work_title">Multi-Post Stories</h1>
+            <ul class="work_info">
+              <li class="w_s">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+            <p class="work_details_content">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work_cats">
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+      </section>
+      <!-- end of work section-->
+
+      <section class="about">
+        <h1 class="headline_title">About Myself</h1>
+        <p class="about_details">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <h3 class="social_title blue">LET’S CONNECT</h3>
+        <ul class="social_links">
+          <li class="social_link">
+            <a href="#"
+              ><img
+                class="social_icon"
+                src="./images/twitter.svg"
+                alt="Twitter"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img
+                class="social_icon"
+                src="./images/linkedin-icon.svg"
+                alt="Linkedin"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img class="social_icon" src="./images/m.svg" alt="M"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img class="social_icon" src="./images/github.svg" alt="Github"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img class="social_icon" src="./images/doit.svg" alt="Diot"
+            /></a>
+          </li>
+        </ul>
+        <button type="button" class="default_btn">Get My Resume</button>
```

Comment:
- [ ] Kindly fix that.
- [ ] Also consider using the same `font-family`
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198302384-03b5bea3-58f4-4b0e-a9d6-ab707813affe.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198302579-36b62c8d-ca76-4a2e-9b6e-06bb1f5d5ea8.png)